source/database/create-database.rst

```diff
@@ -37,7 +37,9 @@ options, these include:
 * You will also need to source an OpenRC file in the correct region
 * It is also necessary to have an **existing network**,  which is attached to a
   router and has a working subnet, on the project that you wish to deploy the
-  database instance to.
+  database instance to. The network that you create the database instance in
+  **must** have access to the internet, as it is required to download the
+  support files to start the database.
```

Comment:
Not only start, but also an image is pulled each time a backup is initiated.

Probably more detail than this bullet point needs, though.